python_modules/dagster/dagster/_core/system_config/composite_descent.py

```diff
@@ -16,82 +18,95 @@
 from dagster._core.system_config.objects import OpConfig
 from dagster._utils.merger import merge_dicts
 
+RawNodeConfig: TypeAlias = Mapping[str, object]
+
 
 class OpConfigEntry(
     NamedTuple("_SolidConfigEntry", [("handle", NodeHandle), ("solid_config", OpConfig)])
 ):
-    def __new__(cls, handle: NodeHandle, solid_config: OpConfig):
+    def __new__(cls, handle: NodeHandle, op_config: OpConfig):
         return super(OpConfigEntry, cls).__new__(
             cls,
             check.inst_param(handle, "handle", NodeHandle),
-            check.inst_param(solid_config, "solid_config", OpConfig),
+            check.inst_param(op_config, "solid_config", OpConfig),
         )
 
 
+# This is a dummy handle used to simplify the code, corresponding to the root container (graph). It
+# doesn't actually represent a node during execution.
+_ROOT_HANDLE = NodeHandle("root", None)
```

Comment:
add some symbols that violate op/graph/asset name rules so theres no chance of collision 

maybe use `is` instead of `==` checks